### ISSUE TITLE
cv2 whitelist not work

### ISSUE THREAD

**Issue by wqh17101** [PyCQA/pylint#5442]
### Bug description

cv2 whitelist not work

### Command used

```shell
pylint a.py --extension-pkg-whitelist=cv2
```

### Pylint output

```shell
E1101: Module 'cv2' has no 'ORB' member (no-member)
E1101: Module 'cv2' has no 'resize' member (no-member)
```

### Expected behavior

do not check cv2

### Pylint version

```shell
pylint 2.11.1
astroid 2.8.6
Python 3.8.8 (default, Apr 13 2021, 15:08:03) [MSC v.1916 64 bit (AMD64)]
```

**Comment by Pierre-Sassoulas** [comment]
``--extension-pkg-allow-list`` means that extensions are loaded into the active Python interpreter and may run arbitrary code, which you may not want for untrusted code. It just tell pylint that cv2 is trusted. It does not means that it won't check cv2 at all or report error for it. See https://pylint.pycqa.org/en/latest/technical_reference/c_extensions.html

**Comment by wqh17101** [comment]
```
import logging
from functools import wraps, partial


class Decorator:

    @staticmethod
    def timing(func=None, *, logger=None):
        print(func)

        if func is None:
            return partial(Decorator.timing, logger=logging)

        @wraps(func)
        def wrapper(*args, **kwargs):
            func_name = func.__name__
            print(func_name)
            res = func(*args, **kwargs)
            return res

        return wrapper


class About:
    """
    About
    """

    def __init__(self):
        self.x_1 = 1

    @Decorator.timing()
    def run(self, flag=False):
        print(flag)
        self.func()

    def func(self):
        print(self.x_1)


if __name__ == '__main__':
    A = About()
    A.run()

```

```
33:4: W0211: Static method with 'self' as first argument (bad-staticmethod-argument)
```
